source/Cosmos.Common/StringHelper.cs

```diff
@@ -410,6 +410,36 @@ internal enum StringComparisonResultEnum {
             }
             return (int)StringComparisonResultEnum.Equal;
         }
+
+        /// <summary>
+        /// Check if string is numeric.
+        /// </summary>
+        /// <param name="aString">A string to check if numeric.</param>
+        /// <returns>Returns TRUE if string is numeric.</returns>
+        public static bool IsNumeric(string aString) {
+            for (int i = 0; i < aString.Length; i++) {
+                if (!char.IsDigit(aString[i])) {
+                    if (i == 0 && aString[i].Equals('-')) {
```

Comment:
Bytes should always be positive, so I dont think we have to check for minus.

Reply:
Done